### PR TITLE
Updates for running ChatQnA + Conversational UI on Gaudi

### PR DESCRIPTION
**Pull request by yogeshmpandey**
## Description

The summary of the proposed changes as long as the relevant motivation and context.

## Issues

List the issue or RFC link this PR is working on. If there is no such link, please mark it as `n/a`.

n/a

## Type of change

List the type of change like below. Please delete options that are not relevant.

- [ ] Bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds new functionality)
- [ ] Breaking change (fix or feature that would break existing design and interface)

## Dependencies

List the newly introduced 3rd party dependency if exists.

## Tests

Describe the tests that you ran to verify your changes.

### CODE REVIEW

**Comment by chensuyue** [comment]
The gaudi driver in CI node has been upgraded into 1.16.1, which is not adapt to current tgi server, we will switch to another test machine at first.

**Review by chensuyue — APPROVED** [review]

**Review by ganesanintel — APPROVED** [review]

File: ChatQnA/docker/gaudi/docker_compose.yaml [hunk 1]
```diff
@@ -195,6 +195,20 @@ services:
     ipc: host
     restart: always
 
+  chaqna-gaudi-conversation-ui-server:
```
Comment (lvliang-intel) [inline ChatQnA/docker/gaudi/docker_compose.yaml:198]: For the "docker_compose.yaml" file, only one UI image is needed. You can mention in the README that users can modify the UI image in the "docker_compose.yaml" file and choose either "opea/chatqna-ui" or "opea/chatqna-conversation-ui".

File: ChatQnA/docker/gaudi/README.md [hunk 2]
```diff
@@ -263,3 +281,23 @@ To access the frontend, open the following URL in your browser: http://{host_ip}
 ```
 
 ![project-screenshot](../../assets/img/chat_ui_init.png)
+
+Here is an example of running ChatQnA:
+
+![project-screenshot](../../assets/img/chat_ui_response.png)
+
+## 🚀 Launch the Conversational UI (react)
```
Comment (lvliang-intel) [inline ChatQnA/docker/gaudi/README.md:289]: Displaying two UIs here might confuse users. It's necessary to include a description stating that users only need to choose one of them.

File: ChatQnA/docker/gaudi/docker_compose.yaml (hunk 1 above)
Comment (yogeshmpandey) [reply ChatQnA/docker/gaudi/docker_compose.yaml]: Agreed, Done.

File: ChatQnA/docker/gaudi/README.md (hunk 2 above)
Comment (yogeshmpandey) [reply ChatQnA/docker/gaudi/README.md]: Done

**Review by WenjiaoYue — APPROVED** [review]

**Pull request merged**